DOCKERFILE

```diff
@@ -0,0 +1,25 @@
+ARG swift_version=5.8
+ARG ubuntu_version=jammy
+ARG base_image=swift:$swift_version-$ubuntu_version
+FROM $base_image
+# needed to do again after FROM due to docker limitation
+ARG swift_version
+ARG ubuntu_version
+
+# set as UTF-8
+RUN apt-get update && apt-get install -y locales locales-all
+ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+
+# Install etcd client
+ARG ETCD_VERSION=v3.4.2
+RUN curl -L https://github.com/etcd-io/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz -o etcd-${ETCD_VERSION}-linux-amd64.tar.gz && \
+    tar xzvf etcd-${ETCD_VERSION}-linux-amd64.tar.gz && \
+    mv etcd-${ETCD_VERSION}-linux-amd64/etcdctl /usr/local/bin/etcdctl && \
+    rm -rf etcd-${ETCD_VERSION}-linux-amd64 etcd-${ETCD_VERSION}-linux-amd64.tar.gz
```

Comment:
This docker image shouldn't contain any etcd related things. Rather it is used for building our package in docker. Can you just copy the setup from https://github.com/swift-server/swift-memcache-gsoc/tree/main/docker but just do swift 5.10 and main as docker compose files. You also need to change the `docker-compose` to pull the etcd server docker image instead of the memcache one.

Reply:
@FranzBusch Updated the dockerfile! Quick question about the docker-compose, mine currently gets the etcd image, and the memcache docker-compose has test / shell commands. Should I update the docker-compose once the etcd client has tests, and leave it as is for now?